kie-wb-common-react/boxed-expression-editor/src/components/Resizer/Resizer.css

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.expression-container .react-resizable.height-based-on-content {
+  height: 100% !important;
```

Comment:
Generally, it's a better idea to use more specific CSS selector instead of using the `!important` card, but if `!important` is really required, can we leave a comment clarifying why? 🙃

Reply:
In general, I agree with you.
For this specific case, it seems that `react-resizable` is always expecting an height, even if we are interested in having 100% for it. Since `react-resizable` specifies inline style, the only way I found to overwrite it, was to assign a specific class (`height-based-on-content`) with the `!important` card.
I am going to update the comment!